Utility.js

```diff
@@ -164,8 +164,16 @@ export function lookUpExpectedPutts(proxHole) {
 }
 
 export function getIconByKey(key) {
-  const icons = [{ carry: "arrow-up" }, { sideLanding: "arrow-left-right" }];
+  const icons = [
+    { carry: "arrow-up" },
+    { sideLanding: "arrow-left-right" },
+    { strokes: "golf-tee" },
+  ];
 
   const iconObject = icons.find((icon) => icon[key]);
   return iconObject ? iconObject[key] : null;
 }
+
+export const getCombinedDrillTitle = (drillInfo) => {
+  return `${drillInfo.prettyDrillType} | ${drillInfo.subType}`;
+};
```

Comment:
Are we considering changing this so that the drill and the subtype are stacked? I am good either way but might not need this if we are going this other although I am unsure if getCombinedDrillTitle is used in other places besides the drill screen. 
ex:
 **20 Shot Challence**
Red Zone

Reply:
nevermind, saw it somewhere else during the review lol

Reply:
If it's stacked, I don't think it should be in a Utility function. I think all drillCards should be the same component

Reply:
If we are stacking, the function isn't needed. This was before we decided that and I have not had the chance to mess around with it yet